src/lib/Components/Inbox/Conversations/Messages.tsx

```diff
@@ -150,7 +149,7 @@ export default createPaginationContainer(
           name
           initials
         }
-        messages(first: $count, after: $after, sort: DESC) @connection(key: "Messages_messages") {
+        messages(first: $count, after: $after, sort: DESC) @connection(key: "Messages_messages", filters: []) {
```

Comment:
Found this in https://facebook.github.io/relay/docs/pagination-container.html#connection-directive

> Particularly, if you write @connection(key: ..., filters:[]), Relay will ignore all the variables.